migrations/v4.js

```diff
@@ -0,0 +1,75 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+const getCourse = content => {
+  const course = content.find(({ _type }) => _type === 'course');
+  return course;
+};
```

Comment:
As of the latest release of adapt-migrations we now have the getCourse function added to helpers so we can now import it

```suggestion
import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse } from 'adapt-migrations';
import _ from 'lodash';
```